src/Language/Fixpoint/SortCheck.hs

```diff
@@ -1131,7 +1131,7 @@ unifySorts :: Sort -> Sort -> Maybe TVSubst
 --------------------------------------------------------------------------------
 unifySorts   = unifyFast False emptyEnv
   where
-    emptyEnv = const $ die $ err dummySpan "SortChecl: lookup in Empty Env "
+    emptyEnv x = die $ err dummySpan $ "SortChecl: lookup in Empty Env: " <> pprint x
```

Comment:
This module is SortCheck. Is SortChecl a typo?

Reply:
yes!

Reply:
Fixed.